### ISSUE TITLE
Angelscript questions.

### ISSUE THREAD

**Issue by echads** [SamVanheer/Angelscript#1]
I see you are well on the way to SharpLife and let me say "finally C# solution for steam" bravo!

We are on way to releasing our first Steam game using valve tools we are stong in Unity but wanted to do valve way first for the community sake.

We are tempted (very) to use your solutions here, as the C# path has easy ties to our Unity upsteam.

Why I am posting here is, there is no Source 2013 base C# that we can find like your work here for halflife 1 goldsource. We do plan to slowly move everything that way, or perhaps do a second release that way. So having a similar modding experience between halflife1 and the destination Source 1 2013 and finally Source 2 in SteamVR Home form. We ask...

TLDR:
Is Angelscript closer to Lua or Python, it seems it is most list JavaScript?

**Comment by SamVanheer** [comment]
Angelscript's syntax is very similar to C#, but it has elements of C++ as well.

Out of the languages you listed JavaScript is the closest to its syntax.

**Comment by echads** [comment]
Neat, we will double check the licenses but I assume your hope is folks use these repos for Steam games and workshop support? Because that is our plan :-) slowly but surely you've done a great thing here.

**Comment by SamVanheer** [comment]
SharpLife uses the same license as the Half-Life SDK. Angelscript isn't made by me so you'll need to check its website for more information.
